felix/dataplane/linux/bpf_ep_mgr.go

```diff
@@ -804,6 +804,18 @@ func (m *bpfEndpointManager) markExistingWEPDirty(wlID proto.WorkloadEndpointID,
 	}
 }
 
+func (m *bpfEndpointManager) syncIfStateMap() {
+	m.ifStateMap.IterDataplaneCache(func(k ifstate.Key, v ifstate.Value) {
+		ifindex := int(k.IfIndex())
+		_, err := net.InterfaceByIndex(ifindex)
+		if err != nil {
+			m.ifStateMap.DeleteDesired(k)
```

Comment:
Since this is a one-off, I'd log this at Info.  Also, worth doing this so we'll have something to go on if we ever get an unexpected error here:
```
if err != nil {
  if os.IsNotExists() {  // Or whatever the right check is
    log.Info...
  } else {
    log.WithError(err).Error(...
  }
```